docs/dev/setup.md

```diff
@@ -0,0 +1,430 @@
+
+    helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+
+    helm search service-catalog
+
+    kubectl create clusterrolebinding tiller-cluster-admin  --clusterrole=cluster-admin --serviceaccount=kube-system:default
+
+    kubectl -n kube-system patch deployment tiller-deploy -p '{"spec": {"template": {"spec": {"automountServiceAccountToken": true}}}}'
+
+    helm install svc-cat/catalog  --name catalog --namespace catalog
+    ```
+
+>**Note:** In order to reuse the docker daemon and speed up local development check the [Reusing the Docker daemon section of the minikube getting started guide](https://kubernetes.io/docs/getting-started-guides/minikube/#reusing-the-docker-daemon)
+
+___
+
+## Setup Service Manager Components
+
+This step shows how to setup the Service Manager as well as the Service Manager Proxies (CF and K8S). There are a couple of ways things can be done.
+For development purposes, the easiest way is to have everything (Service Manager, CF Proxy, K8S Proxy, Dummy Broker) to be running locally on your machine and have them configured to talk to Minikube and PCF Dev. Alternatively, the components can be setup to run on PCF Dev or Minikube and you may run on `localhost` only the component that you are currently working on.
+
+The next steps describe how each component can be setup either to run locally or to run on a locally prepared platform (local CF / K8S minikube)
+
+### Setup the Service Manager
+
+Clone the repository:
+
+```bash
+git clone https://github.com/Peripli/service-manager.git
+
+```
+
+#### Run Service Manager on localhost
+
+>**Prerequisites:**
+>
+>Setup Postgresql
+>
+>In order for the Service Manager to run locally, itrequires a >Postgresql DB.
+>
+> * Install on your machine
+>
+>    Windows users may follow [this guide]>>(http://www.postgresqltutorial.com/install-postgresql/) in order to install postgresql.
+>
+> * Spin up a postgres docker container
+>
+>    ``` bash
+>    docker run --name postgres-docker -d -p 5432:5432 -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres postgres
+>    ```
+>
+>The DB should be accessible on the ip that the docker machine is running on (docker-machine ip) and port 5432.
+
+* Modify the contents of application.yml to point to the configured Postgresql
+
+    ``` yml
+    server:
+      requestTimeout: 3000
+      shutdownTimeout: 3000
+      port: 8080
+    log:
+      level: debug
+      format: text
+    db:
+      name: sm-postgres
+      uri: postgres://postgres:postgres@{postgres-ip}:5432/postgres?sslmode=disable
+    ```
+
+    >**Note**: Modify `server.port` to specify a different port on which the Service Manager should run.
+
+* Modify the contents of `service-manager/api/osb/logic.go`
+
+    ```go
+    func (b *BusinessLogic) ValidateBrokerAPIVersion(version string) error {
+        return nil
+    }
+    ```
+
+* Navigate to the `service-manager` folder and run the application:
+
+    ``` go
+    go run cmd/main.go

```

Comment:
go run cmd/cf/main.go

Reply:
service-manager i028667$ go run cmd/cf/main.go
FATA[0000] Error creating the serverError loading configuration: unexpected end of JSON input
exit status 1

Reply:
if you use cmd/cf/main.go you would need to comment line https://github.com/Peripli/service-manager/blob/master/cmd/cf/main.go#L35 and replace it with

```
srv, err := sm.NewServer(ctx, env.Default())
```

Reply:
It could be just main.go (in order to be deployed to cf without docker)